doc/content/whats-new/3.22.2.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 
```

Comment:
This can already be November 10.

https://github.com/TheThingsNetwork/lorawan-stack/releases/tag/v3.22.2
https://github.com/TheThingsIndustries/lorawan-stack/releases/tag/v3.22.2